docs/configuration/secrets/troubleshoot.md

```diff
@@ -0,0 +1,35 @@
+# Troubleshooting
+
+The sections below include information to help you troubleshoot issues that you may come across while configuring the secrets used by Codacy.
+
+If the information provided on this page is not enough to solve your issue, contact [support@codacy.com](mailto:support@codacy.com) providing all the information that you were able to collect while following the troubleshooting instructions.
+
+## Lost/Changed initially generated secrets
+
+While trying to open Codacy,
+you were shown an error message stating that,
+the secret on the database and the one in your configuration file are different.
+
+### Steps
+
+1. Get the latest logs from the API service
```

Comment:
Since the script automatically extracts the value from the logs, we can simplify this a bit:

```suggestion
1. Obtain the correct key used to encrypt sensitive data in the database from the Codacy logs.
```